generators/pdf_to_text/pdf_to_text/__init__.py

```diff
@@ -0,0 +1,28 @@
+from pydantic import BaseModel
+from PyPDF2 import PdfReader
+
+
+
+INPUT_EXAMPLE = {
+    "pdf": "",
+}
+
+
+class PdfToTextModel(BaseModel):
+    pdf: str
+
+    class Config:
+        schema_extra = {"example": INPUT_EXAMPLE}
+
+
+def pdf_to_text(req:PdfToTextModel):
+    pdf = req.pdf
+    reader = PdfReader(pdf)
+    number_of_pages = len(reader.pages)
+    number = 0
+    output = ""
+    while number < number_of_pages:
+        output += "".join(reader.pages[number].extract_text())
+        number +=1
+    return output
```

Comment:
Not too sure how that would work out, as we cannot really send PDF documents to the endpoint? Please correct me if I'm wrong